GOVERNANCE.md

```diff
@@ -0,0 +1,84 @@
+# Governance document for Epiverse-TRACE
+
+## Acronyms
+
+- TRACE: Tools for Response, Analytics, and Control of Epidemics
+- LSHTM: London School of Hygiene and Tropical Medicine 
+- MRCG: Medical Research Council Unit, Gambia
+- LAC: Latin America and the Caribbean
+
+## Team Structure and Roles
+
+The Epiverse-TRACE GitHub organization contains 3 teams, representing the 3 partner organizations currently involved as equals in the project:
+
+- data.org
+- LSHTM-MRCG
+- TRACE-LAC (UniAndes & Javeriana)
+
+Each team has [organization owners](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#organization-owners), who can add new individuals to the GitHub organization, administrator access to individual repositories, and edit the organization-level settings. [The organization owners are](https://github.com/orgs/epiverse-trace/people?query=role%3Aowner):
+
+- Bubacarr Bah
+- Zulma Cucunubá
+- Roz Eggo
+- Sebastian Funk
+- Hugo Gruson
+- Chris Hartgerink
+- Adam Kucharski
+- Edwin van Leeuwen
+- Jaime Pavlich-Mariscal
```

Comment:
```suggestion
- Adam Kucharski
- Bubacarr Bah
- Chris Hartgerink
- Edwin van Leeuwen
- Hugo Gruson
- Jaime Pavlich-Mariscal
- Roz Eggo
- Sebastian Funk
- Zulma Cucunubá
```

Alternatively alphabetical sort if there's no particular reason for the order.

Reply:
It is alphabetical but on the last name. No idea which one is preferable.

Reply:
Feel free to ignore - not important enough and I did not notice that before 😊